### DISCUSSION TITLE
How can I publish or make notes Public? Also, Can I EMBED the notes (using iframe/oembed) elsewhere?

### DISCUSSION THREAD

**Discussion by drjagan** [TriliumNext/Trilium#2010] [Q&A]
Hi, I am a professor and find trilium a very promising and fantastic tool to create study notes. 
I wish I could do the following:
1. Publish the notes into some sort of publically accessible site
2. Embed the notes into our LMS (Canvas or Moodle) using either ordinary iframe or Oembed. 

Can these be done? If so, can you be so kind enough to let me know how?

If these can't be done, can I add these as feature requests please? 

Thank you,
Cheers

**Comment by zadam** [comment]
Hi, publishing notes online is not currently supported, although it is a relatively high priority item on the TODO list. I would give ETA around end of the year.
